src/main/webapp/app/exam/exam-scores/exam-scores.component.ts

```diff
@@ -112,6 +113,7 @@ export class ExamScoresComponent implements OnInit {
             typeof studentResult.overallPointsAchieved === 'undefined' || studentResult.overallPointsAchieved === null ? '' : round(studentResult.overallPointsAchieved, 1);
         csvRow.overAllScore =
             typeof studentResult.overallScoreAchieved === 'undefined' || studentResult.overallScoreAchieved === null ? '' : round(studentResult.overallScoreAchieved, 2);
+        csvRow.submitted = studentResult.submitted ? 'yes' : 'no';
```

Comment:
That's okay for me, just as a thought: Do we want to translate it?

Reply:
Thanks for the feedback! Atm the headers are also not translated therefore I would ignore it here too